### PR TITLE
chore: v3.0.0-beta2 release

### PR DESCRIPTION
**Pull request by vrundakansara**
- New: Visibility options - Now user can hide few options as per devices.
- New: Divider Element - Horizontal & Vertical Divider for both Header and Footer
- New: WooCommerce Cart and EDD Cart Element for Header
- Improvement: Faster Customizer experience with respect to Design/General Tab change.
- Improvement: Ability to add any number Elements in Header and Footer using a filter from child theme.
- Improvement: Loaded animation CSS only when needed.
- Improvement: Improved the UI in customizer for Footer Builder.
- Improvement: Custom Layouts - Added compatibility to new Custom Layouts Hooks with respect to the new Header/Footer.
- Improvement: Page Header - Added compatibility to Page Headers with respect to the new Header/Footer.
- Improvement: Responsive - Added responsive compatibility to Footer Inner Column Spacing.
- Improvement: Responsive - Added responsive compatibility to Social Icon Size and Social Icon Spacing.
- Improvement: Responsive - Added responsive compatibility to Search Icon Color.
- Improvement: Responsive - Added responsive compatibility to HTML Element related Colors.
- Improvement: Responsive - Added responsive compatibility to Widget Element related Colors.
- Improvement: Responsive - Added responsive compatibility to Above, Below and Primary Header height.
- Improvement: Responsive - Added responsive compatibility to Footer Menu layout.
- Improvement: Responsive - Added responsive compatibility to Footer Column Layout.
- Fix: Allowed negative margin values
- Fix: Fixed the issue with Global Button hover color not being applied to Header/Footer Buttons.
- Fix: Sticky Header - Shrink option issue with alignment.
- Fix: Primary Menu - Font Weight not being applied in frontend.
- Fix: Off Canvas - Fixed issue with Sub Menu colors, typography and other inheriting properties.
- Fix: Primary Menu - Fixed an issue with Stack on Mobile option not in action in some rare scenario.
- Fix: Footer - Added default Stack layout for Footer Columns.
- Fix: Toggle Button - Fixed an issue Toggle Button SVG looking odd.
- Fix: Button - Fixed an issue with mismatched EM/PX with respect to Margin.
- Fix: AMP Compatibility - Fixed an issue with Menu animation JS for AMP pages.
- Fix: Slider Control - Fixed an issue with zero value not being accepted by the Slider Control.

### CODE REVIEW

**Review by imnavanath — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

File: inc/compatibility/lifterlms/class-astra-lifterlms.php [hunk 1]
```diff
@@ -185,7 +185,7 @@ public function customize_register( $wp_customize ) {
 			 */
 			require ASTRA_THEME_DIR . 'inc/compatibility/lifterlms/customizer/sections/class-astra-lifter-container-configs.php';
 			require ASTRA_THEME_DIR . 'inc/compatibility/lifterlms/customizer/sections/class-astra-lifter-sidebar-configs.php';
-			require ASTRA_THEME_DIR . 'inc/compatibility/lifterlms/customizer/sections/layout/class-astra-lifter-general-configs.php';
+			// require ASTRA_THEME_DIR . 'inc/compatibility/lifterlms/customizer/sections/layout/class-astra-lifter-general-configs.php';
```
Comment (vrundakansara) [inline inc/compatibility/lifterlms/class-astra-lifterlms.php:188]: @premanshup - Can you confirm this once?

**Review by rohitp-bsf — APPROVED** [review]
LGTM

File: inc/compatibility/lifterlms/class-astra-lifterlms.php (hunk 1 above)
Comment (premanshup) [reply inc/compatibility/lifterlms/class-astra-lifterlms.php:188]: @rohitp-bsf Can you please check this?

Comment (rohitp-bsf) [reply inc/compatibility/lifterlms/class-astra-lifterlms.php:188]: Yes, We have removed this!

**Review by premanshup — APPROVED** [review]

**Pull request merged**